### ISSUE TITLE
Latex support for annotation comment？

### ISSUE THREAD

**Issue by roachsinai** [burtonator/polar-bookshelf#323]
Thanks for this great work. But will you add latex support for annotation comment?

**Comment by burtonator** [comment]
maybe something like mathquill

http://mathquill.com/

**Comment by sandersantema** [comment]
Maybe instead of latex support add mathjax support just like anki implemented it. If you support the same delimiters (if that's what they're called?) \[...\] and \(...\) you can directly transfer these annotations from polar to Anki. Of course full latex support is nice but if you're going for full compatibility with Anki mathjax seems the way to go.

**Comment by burtonator** [comment]
don't they all use latex internally?  mathjax+katex just being libraries that support rendering latex?

**Comment by samuelcwl** [comment]
Upvoting this - this will be huge (esp. with full compat. with Anki).

**Comment by burtonator** [comment]
Agreed... it will happen eventually but right now not enough funding to land all the feature requests we would like.

**Comment by vonHartz** [comment]
Any update on this? Couldn't find it in the roadmap Q1 2020.

**Comment by desk7** [comment]
Any update?
